repo2docker/buildpacks/base.py

```diff
@@ -71,6 +71,9 @@
 {% endfor -%}
 {% endif -%}
 
+# So users can move executables here with postBuild, issue #557
+ENV PATH {{ /home/jovyan/{local}/bin:${PATH}}}
```

Comment:
I think this needs to be `'/home/jovyan/local/bin'` instead of `/home/jovyan/{local}/bin`.